README.md

```diff
@@ -72,11 +71,83 @@ steps in [contributing.md](contributing.md).
 
 Wait for approval of the Pull Request and merge the change.
 
-## Support
+### Code quality
+
+Code style guidelines ease understanding and maintaining code. Automated checks
+enforce some of the guidelines.
+
+#### Python
+
+*   Code must be formatted according to the
+    [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
+    according to the [yapf formatter](https://github.com/google/yapf).
+*   Code must not generate lint errors or warnings according to
+    [pylint](https://www.pylint.org/) configured for the
+    [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
+    as specified in `.pylintrc`.
+*   Tests must succeed.
+
+Consider automating coding to satisfy some of these requirements.
+
+*   [Integrate pylint](http://pylint.pycqa.org/en/latest/user_guide/ide-integration.html)
+    with your editor.
+*   Integrate yapf with
+    [Visual Studio](https://code.visualstudio.com/docs/python/editing#_formatting),
+    [Emacs](https://www.emacswiki.org/emacs/PythonProgrammingInEmacs#toc21),
+    [vim](https://github.com/mindriot101/vim-yapf#why-you-may-not-need-this-plugin).
+    Specify the Google style using `--style google`.
+
+To run the tools via a command line:
 
-For general questions or issues about importing data into Data Commons,
-please open an issue on our [issues](https://github.com/datacommonsorg/data/issues) page. For all other
-questions, please send an email to `support@datacommons.org`.
+*   [pylint](http://pylint.pycqa.org/en/latest/user_guide/run.html).
+*   After [installing yapf](https://github.com/google/yapf#id2), execute using
+    `--style google`, e.g.,
+
+```
+# Update (--in_place) all files in the util/ directory and its subdirectories.
+yapf --recursive --in_place --style google util/
+
+# Produce differences between the current code and reformatted code.  Empty
+# output indicates correctly formatted code.
+yapf --recursive --diff --style google util/
+```
+
+To run a unit test, use a command like
+
+```
+python3 -m unittest discover -v -s util/ -p "*_test.py"
+```
+
+The `discover` option searches (`-s`) the `util/` directory for files with
+filenames ending with `_test.py`. It considers all these files to be unit tests
+to be run. Output is verbose (`-v`).
+
+#### Disabling style checks
+
+Occasionally, one has to disable style checking or formatting for particular
+lines.
+
+To
+[disable pylint for a particular line or block](http://pylint.pycqa.org/en/latest/user_guide/message-control.html)
+, use syntax like
+
+```
+# pylint: disable=line-too-long,unbalanced-tuple-unpacking
+```
+
+To disable yapf for some lines,
+
+```
+# yapf: disable
+... code ...
+# yapf: enable
+```
+
```

Comment:
Add a TODO to add the equivalent docs for Go code

Reply:
Done.